libnmstate/sysctl.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import errno
+
+
+def enable_ipv6(dev):
+    disable_ipv6(dev, disable=False)
+
+
+def disable_ipv6(dev, disable=True):
+    try:
+        with open('/proc/sys/net/ipv6/conf/%s/disable_ipv6' % dev, 'w') as f:
```

Comment:
This should use .format() IMHO, since we use it everywhere else, too.

Reply:
Many like using the old stile for a *single* variable scenario, it is shorter and cleaner.
If you insist on this, I can change it (let me know).